### ISSUE TITLE
Spread operator issues

### ISSUE THREAD

**Issue by fluffynuts** [trullock/NUglify#232]
Referring to https://github.com/trullock/NUglify/issues/84 : if I recreate that scenario inside a test, whilst the output is minified, the result has `HasErrors` set to true, and has the error `Implicit property name must be identifier: ...obj1`; this is using nuget package NUglify 1.13.2

That issue is marked as resolved - so what am I doing wrong in the following code?:

```csharp
using NExpect;
using NUglify;
using NUglify.JavaScript;
using NUnit.Framework;
using static NExpect.Expectations;

namespace Tests
{
	public class MinifyJavascript
	{
            [Test]
            public void ShouldHandleSpreadOperator()
            {
                // Arrange
                var script = @"
const obj1 = {
    prop0: 'val0',
    prop1: 'val1'
};
const obj2 = {
    ...obj1,
    prop2: 'val2'
};
";
                // Act
                var settings = new CodeSettings()
                {
                    ScriptVersion = ScriptVersion.EcmaScript6
                };
                var result = Uglify.Js(script, settings);
                // Assert
                Console.Error.WriteLine(result.Code);
                Expect(result.HasErrors)
                    .To.Be.False(() => string.Join(
                            "\n", result.Errors.Select(e => e.Message)
                        )
                    );
                Expect(result.Code)
                    .Not.To.Contain("errors");
            }
	}
}
```

**Comment by trullock** [comment]
Does it minify correctly?

**Comment by fluffynuts** [comment]
It does appear to - but still marks the result as failed and adds in error messages, so for places where a kind of "try to minify, otherwise return the original script" strategy is followed, the overall result is failure.

**Comment by trullock** [comment]
OK I'll investigate, #84 perhaps didnt tidy up the error

**Comment by trullock** [comment]
I can repro this.

I really dont like the test framework inherited with NUglify, I suspect theres others errors like this.

The primary test runner doesn't complain if it finds errors like this.

Will investigate fixing that at the same time, but I'll try not to hold this fix up

**Comment by trullock** [comment]
Fixed in 1.13.3

**Comment by fluffynuts** [comment]
Thanks, this is working as expected 👍
